en/theme/material/assets/css/versions.css

```
@@ -0,0 +1,21 @@
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+.md-content__inner {
```

Stylelint (17.2.0)

[error] Expected class selector ".md-content__inner" to be kebab-case (selector-class-pattern)

(selector-class-pattern)